PowerEditor/src/ScitillaComponent/ScintillaEditView.cpp

```diff
@@ -139,6 +139,8 @@ LanguageName ScintillaEditView::langNames[L_EXTERNAL+1] = {
 {TEXT("json"),			TEXT("json"),				TEXT("JSON file"),										L_JSON,			SCLEX_CPP },
 {TEXT("javascript.js"), TEXT("JavaScript"),			TEXT("JavaScript file"),								L_JAVASCRIPT,	SCLEX_CPP },
 {TEXT("fortran77"),		TEXT("Fortran fixed form"),	TEXT("Fortran fixed form source file"),					L_FORTRAN_77,	SCLEX_F77},
+{TEXT("less"),			TEXT("Less"),				TEXT("Less File"),                                      L_LESS,			SCLEX_CSS },
+{TEXT("scss"),			TEXT("SCSS"),				TEXT("SASS File"),						                L_SCSS,			SCLEX_CSS },

```

Comment:
Shouldn't it be "S**C**SS File" or "Sassy CSS File" as SASS syntax is not equal to SCSS syntax?